src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/__artifactId__-servlet.xml

```diff
@@ -25,6 +25,9 @@ ${shogun2-parent-package}.*.web" )
 			serialization -->
 		<mvc:message-converters>
 			<beans:bean
+				class="org.springframework.http.converter.ByteArrayHttpMessageConverter">
+			</beans:bean>
```

Comment:
It automatically sets the correct `Content-Type` header for ResponseEntities of type ByteArray (for all [supported MediaTypes](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/http/MediaType.html)). Without this converter the response send by the Interceptor (and others) couldn't be read correctly by the browser.